flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/TestBaseUtils.java

```diff
@@ -480,16 +480,22 @@ protected static File asFile(String path) {
 				resultStrings[i] = (val == null) ? "null" : val.toString();
 			}
 		}
-		
-		assertEquals("Wrong number of elements result", expectedStrings.length, resultStrings.length);
+
+		//
```

Comment:
Looks like you were intending to add a comment here. The `if (sort) ...` block will need to go before `msg` is created. Should we add newlines to the message string, something like `"Expected %d elements but received %d.\n  expected array: %s\n  received array: %s"`? That should line up.